cmd/govim/textedit.go

```diff
@@ -158,6 +158,8 @@ func (v *vimstate) applyProtocolTextEdits(b *types.Buffer, edits []protocol.Text
 			},
 		},
 	}
+
+	v.triggerBufferASTUpdate(b)
```

Comment:
Small point. Please can we put this on line 149, after the `b.SetContents()` call? It follows more clearly that way.